File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/api/VersionControlAPIs.java
package ir.ayantech.versioncontrol.api;

import androidx.annotation.Nullable;

/**
 * Created by Administrator on 11/5/2017.
 */

public class VersionControlAPIs {
    public static CheckVersion checkVersion;
    public static GetLastVersion getLastVersion;

    public static void initialize(@Nullable String baseUrl) {
        checkVersion = new CheckVersion(baseUrl);
        getLastVersion = new GetLastVersion(baseUrl);
    }
}

File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/api/VersionControlAPIs.kt
package ir.ayantech.versioncontrol.api

object VersionControlAPIs {
    @JvmField
    var checkVersion: CheckVersion? = null

    @JvmField
    var getLastVersion: GetLastVersion? = null

    @JvmStatic
    fun initialize(baseUrl: String?) {
        checkVersion = CheckVersion(baseUrl)
        getLastVersion = GetLastVersion(baseUrl)
    }
}
